bndtools.builder/src/org/bndtools/builder/BndtoolsBuilder.java

```diff
@@ -113,16 +118,57 @@
 					.getDynamicReferences();
 			}
 
-			if (model == null) {
-				try {
-					model = Central.getProject(myProject);
-				} catch (Exception e) {
-					markers.deleteMarkers("*");
+			Project ourModel = null;
+			try {
+				ourModel = Central.getProject(myProject);
+			} catch (Exception e) {
+				markers.deleteMarkers("*");
+				// If the cnf project is missing there will likely be a myriad
+				// of Java problems. These do nothing but clutter the error
+				// output and make it hard to find the real problem in the
+				// problem log.
+				myProject.deleteMarkers(IJavaModelMarker.JAVA_MODEL_PROBLEM_MARKER, true, IResource.DEPTH_INFINITE);
```

Comment:
It does seem a bit presumptuous of us (Bndtools) to remove the error markers of another builder, no? I am not sure we should do this.

Reply:
I suppose i could try and see if we can coerce the JavaBuilder to do it itself under these conditions, eg by failing the BndClasspathContainer somehow.

Reply:
I don't think that is a good idea or possible either. The Java builder's errors are real. They are compilation failures. And they are the responsibility of the Java builder to manage.

Reply:
I disagree, but rather than hold up this PR over this issue (which is not directly related to the original problem)  I have removed the offending lines for now so that we can get it merged. We can discuss this further at another time.